docs-src/typescript/generated/backgroundcheck-boilerplate-workflow-details.md

```diff
@@ -10,7 +10,7 @@ tags:
 ---
 
 <!-- DO NOT EDIT THIS FILE DIRECTLY.
-THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-typescript/blob/main/chapter_durable_execution/backgroundcheck/src/workflows.ts. -->
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-typescript/blob/fixes2/chapter_project_setup/backgroundcheck/src/workflows.ts. -->
 
 Temporal Workflows may have any number of custom parameters. However, we
 strongly recommend that you use objects as parameters, so that the object's
```

Comment:
This might just be me, but for all the different places we've tried to call this out (and move this guidance around), I feel like its inclusion here is a little too brief and unclear. "Use objects as parameters" doesn't even quite get at the rationale here; I'm not sure someone reading this for the first time would understand it as "try to supply all the input to the function as a single parameter (e.g. an object with multiple values), because changing the function's definition can cause determinism errors."

Reply:
1. This gets hit hard over the head with a mallet in multiple places and it is primarily a concern when versioning. So thats when we will spend more time on it.
2. Its not as important as we make it out to be to get started. It just isn't - I can build a huge app and learn a lot about Temporal without fussing about Object based params for versioning considerations - which in the end serves to understand Replay + Production level versioning.

We simply can not go into depth on everything all the time.